z_test.go

```diff
@@ -3315,16 +3315,30 @@ func TestPreFetchQuery(t *testing.T) {
 		pf, as   int
 		srt, mrt uint
 	}{
-		{-1, -1, 1, 2},
-		{0, -1, 2, 2},
-		{1, -1, 2, 2},
-		{2, -1, 1, 2},
-		{100, -1, 1, 1},
-		{-1, 100, 1, 2},
-		{0, 100, 2, 2},
-		{1, 100, 2, 2},
-		{2, 100, 1, 2},
-		{100, 100, 1, 1},
+		{-99, -99, 1, 4},
```

Comment:
Use a named constant for all appearances of -99.

Reply:
incorporated the change...